client/home/lib/addons/components/addondeactivation/view.coffee

```diff
@@ -0,0 +1,47 @@
+React = require 'app/react'
+{ connect } = require 'react-redux'
+addon = require 'app/redux/modules/payment/addon'
+PlanDeactivation = require '../plandeactivation'
+
+class AddOnDeactivation extends React.Component
+
+  constructor: (props) ->
+
+    super props
+
+
+  deactivateBusinessAddOn: () ->
+
+    @props.onDeactivateBusinessAddOn()
+
+
+  render: ->
+
+    <div>
+      {
+        <PlanDeactivation.Container
+          target='BUSINESS ADD-ON'
+          onDeactivation={@bound 'deactivateBusinessAddOn'} />  if @props.addonStatus
+      }
+    </div>
+
+
+  mapStateToProps = (state) ->
+
+    return {
+      addonStatus: addon.isActivated state
+    }
+
+
+  mapDispatchToProps = (dispatch) ->
+
+    return {
+      onDeactivateBusinessAddOn: ->
+        dispatch(addon.remove())
+    }
+
+
+  module.exports = connect(
```

Comment:
view.coffee files should only be responsible for rendering view with given props.
This usage is against the terminology.